draft-ietf-tls-tls13.md

```diff
@@ -3799,6 +3832,14 @@ a string of Hash.length zeroes is used.  Note that this does not mean skipping
 rounds, so if PSK is not in use Early Secret will still be
 HKDF-Extract(0, 0).
 
+Note: There are multiple potential Early Secret values depending on
+which PSK the server ultimately selects.
```

Comment:
You need to be clearer here about what this implies.  With 0-RTT, only one of the the possible values of Early Secret is relevant.  And I believe that is the only relevant factor, since early exporters aren't useful without 0-RTT.

The main implication is that if a client attempts to use 0-RTT but the server either picks a different PSK or rejects PSK, then the Early Secret has to be re-calculated.